tests/cnf/core/network/metallb/tests/bgp-gracefulrestart.go

```diff
@@ -254,3 +475,92 @@ func removeMetalLbCR() {
 	err = metalLbNs.CleanObjects(tsparams.DefaultTimeout, metallb.GetMetalLbIoGVR())
 	Expect(err).ToNot(HaveOccurred(), "Failed to remove MetalLB object from operator namespace")
 }
+
+func updateMachineConfigurationNodeDisruptionPolicy() {
```

Comment:
Could you comment what this function is doing and why is needed
(fyi I am no very familiar with NodeDisruptionPolicy)

Reply:
Added a comment